### ISSUE TITLE
spark bin download error

### ISSUE THREAD

**Issue by palindrom615** [palindrom615/sdkman#1]
Because spark sdk is provided as tar.gz and not zip file, an error occurs.

**Comment by palindrom615** [comment]
You should care about file name!

**Comment by palindrom615** [comment]
spark: 

```
Date: Tue, 20 Aug 2019 16:28:43 GMT
Server: Apache/2.4.7 (Ubuntu)
Last-Modified: Wed, 01 May 2019 05:57:50 GMT
ETag: "db557d9-587cd329a3000"
Accept-Ranges: bytes
Content-Length: 229988313
Content-Type: application/x-gzip
```
java: 

```
x-amz-id-2: 7TK/bKoXjSvMczeiL1m0BisCeKlPGWIDe4zvAIm5Ou3S+X3NYOQIAwc0LQ/NFt5/39Lso3khxLE=
x-amz-request-id: 070C42C0A789907F
Date: Tue, 20 Aug 2019 16:24:31 GMT
Last-Modified: Thu, 18 Jul 2019 12:45:46 GMT
ETag: "30ab446d8434a775c4a880e3f900a7a6"
Content-Disposition: attachment; filename=OpenJDK11U-jdk_x64_windows_hotspot_11.0.4_11.zip
Accept-Ranges: bytes
Content-Type: application/octet-stream
Content-Length: 200239699
Server: AmazonS3
```

**Comment by palindrom615** [comment]
Fixed with commit 17fc7d6
